Polygon_mesh_processing/include/CGAL/Rigid_triangle_mesh_collision_detection.h

```diff
@@ -203,17 +203,7 @@ class Rigid_triangle_mesh_collision_detection
   //! move assignment operator
   Rigid_triangle_mesh_collision_detection& operator=(Rigid_triangle_mesh_collision_detection&& other)
   {
-    m_own_aabb_trees = std::move(other.m_own_aabb_trees);
-    m_aabb_trees = std::move(other.m_aabb_trees);
-    m_is_closed = std::move(other.m_is_closed);
-    m_points_per_cc = std::move(other.m_points_per_cc);
-    m_traversal_traits = std::move(other.m_traversal_traits);
-    m_free_id = std::move(other.m_free_id);
-    m_id_pool = std::move(other.m_id_pool);
-
-    for(std::size_t i = 0; i< other.m_own_aabb_trees.size(); ++i)
-      other.m_own_aabb_trees[i]= false;
-
+    *this = std::move(other);
     return *this;
   }
```

Comment:
No! Obviouly the move-assignment operator cannot be implemented with itself.

Sébastien's suggestion was to implement the move-constructor using the move-assignment.